### DISCUSSION TITLE
cut set label when the label is long

### DISCUSSION THREAD

**Discussion by vongmengty** [upsetjs/upsetjs#92] [Q&A] [answered]
the upsetjs was cut my set label when the label is long.
Can you give me some advise please?
Thanks!

**Screenshots**
![image](https://user-images.githubusercontent.com/19471093/146343554-7b0cd248-8f98-4646-a615-b1c72094b7af.png)

**Comment by sgratzl** [comment, marked as answer]
as far as I can see you already left aligned the labels. In order to increase the overall space you can play around with the `widthRatio` parameter, see https://upset.js.org/api/react/interfaces/upsetlayoutprops.html#widthratios
